src/Twig/Extension/SonataAdminExtension.php

```diff
@@ -137,7 +143,10 @@ public function getFunctions()
         ];
     }
 
-    public function getName()
+    /*
+     * @return string
```

Comment:
This is redundant with the return type hint.

Why phpcs fixer is not complaining...